### PR TITLE
Add NER  training data guide. 

### PR DESCRIPTION
**Pull request by zewditu**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/machine-learning/how-to-guides/ner-dataset-guide.md](https://github.com/dotnet/docs/blob/6d64ab046cb9ddfe912ff76944767b2293fb6175/docs/machine-learning/how-to-guides/ner-dataset-guide.md) | [How to format data for Named Entity Recognition (NER)](https://review.learn.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/ner-dataset-guide?branch=pr-en-us-39643) |

### CODE REVIEW

File: docs/machine-learning/how-to-guides/ner-dataset-guide.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+NER dataset shapes:
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/ner-dataset-guide.md:1]:
```suggestion
---
title: How to format data for Named Entity Recognition (NER)
description: Learn how to format data for Named Entity Recognition (NER) scenario in Model Builder
ms.date: 02/23/2024
author: zewditu
ms.author: zehailem
ms.custom: mvc,how-to
ms.topic: how-to
#Customer intent: As a non-developer, I want to be able to format training data for Model Builder to use for training NER scenarios
---

NER dataset shapes:
```

Comment (luisquintanilla) [reply docs/machine-learning/how-to-guides/ner-dataset-guide.md:1]: Add metadata

**Review by luisquintanilla — COMMENTED** [review]
Add metadata

File: docs/machine-learning/how-to-guides/ner-dataset-guide.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+title: How to format data for Named Entity Recognition (NER)
+description: Learn how to format data for Named Entity Recognition (NER) scenario in Model Builder
+ms.date: 02/23/2024
+author: zewditu
+ms.author: zehailem
+ms.custom: mvc,how-to
+ms.topic: how-to
+#Customer intent: As a non-developer, I want to be able to format training data for Model Builder to use for training NER scenarios
+---
+
```
Comment (luisquintanilla) [inline docs/machine-learning/how-to-guides/ner-dataset-guide.md:11]:
```suggestion
# How to format data for Named Entity Recognition (NER)
 
```

**Review by luisquintanilla — COMMENTED** [review]
Added title

**Review by luisquintanilla — APPROVED** [review]
LGTM. Thank you @zewditu

**Comment by gewarren** [comment]
Related to #39353

**Pull request merged**